templates/account/logout.html

```diff
@@ -5,12 +5,12 @@
 {% block title %}{% trans_html "py.templates.account.logout.title" default="Log out" %}{% endblock %}
 
 {% block content %}
-    <form class="account_form" method="POST" action="{% url "account_logout" %}">
+    <form class="account_form logout" method="POST" action="{% url "account_logout" %}">
```

Comment:
This is https://www.pivotaltracker.com/story/show/175882795 and we won't solve it in this pull request.